extensions/ql-vscode/src/databases/code-search-api.ts

```diff
@@ -16,10 +13,9 @@ export async function getCodeSearchRepositories(
   }>,
   token: CancellationToken,
   credentials: Credentials,
-  logger: NotificationLogger,
```

Comment:
To make this easier to test, I think we should keep using the `logger` (although it can be the `BaseLogger` instead of the `NotificationLogger`) instead of importing `extLogger` directly. I think we're moving away from importing `extLogger` directly and always trying to use a `Logger` interface.